src/release/extensionApi.js

```diff
@@ -59,7 +57,7 @@ module.exports = function extensionApiFactory(params = {}, data = { extension: {
          * @param {Object} data
          */
         setData(newData) {
-            data = newData;
+            Object.assign(data, newData);
```

Comment:
This implementation doesn't need to change.

Reply:
Сhanges removed.
The initial problem is release process were failed on `selectExtension` step.
The reason is we're trying to read value from `data.extension.name` where `data.extension` is undefined
We don't have `data.extension` because we overdid default `data` by calling `setData`.
There are 3 possible solutions:
1. Fix condition in  and extend config data with `extension` filed in `selectExtension` method.
2. Define default config values and merge with new data in `setData`
3. We call `setData` from `loadConfig` with override config (without `extension` field). Fix `loadConfig`
Could you suggest what is the best way to fix it?

Reply:
You are right, I was wrong!
Let's stick with option 2, what you did before I commented.